plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -152,7 +152,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                 return false;
 
             case SUPPORTS_COMMENT_ON_VIEW_COLUMN:
-                return false;
+                return true;
```

Comment:
`TestDeltaLakeConnectorTest` runs only on file metastore and it doesn't cover other metastore (HMS & Glue). Please add test to `BaseConnectorSmokeTest`.

Reply:
Thank you for your reminder! I am not familiar with the related codes. 🙇